odoo/addons/base/res/res_currency_view.xml

```diff
@@ -11,18 +11,6 @@
             </field>
         </record>
 
-        <record id="view_currency_rate_form" model="ir.ui.view">
```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/odoo/addons/base/res/res_currency_view.xml#L38-L58